docs/DeveloperGuide.md

```diff
@@ -302,28 +302,169 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - List all contacts**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User issues the list command.
+2.  The system retrieves and displays the list of all contacts to the user.
+
+    Use case ends.
+
+
+
+**Use case: UC02 - Add a contact**
+
+**MSS**
+
+1.  User issues the add-contact command with the corresponding details.
+2.  The system adds the contact and displays a success message.
+3.  The system shows the new contact in the address book.
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. The system detects a name input error (duplicated or trailing whitespace).
+    * 1a1. The system displays an error message.
+  
+        The use case ends.
```

Comment:
Maybe we can standardise `Use case ends.` for all instead of `The use case ends.`?

Reply:
Sure, I'll make the changes, thanks